lib/vcenter.json

```diff
@@ -0,0 +1,3404 @@
+{
```

Comment:
if we are gonna keep this approach lets's put it a folder called `data` or something, being in lib and not being a lib does not feel right to me.